### PR TITLE
Fix: Add  Table and  fix small issue  

### PR DESCRIPTION
**Pull request by hardiktheprogrammer**
### Description of the Changes

Add table in bridged tokens and fix small issue

### Check List

- [x] Changes have been done against master branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1057)

### CODE REVIEW

**Comment by hardiktheprogrammer** [comment]
@stoobie  👀

File: components/Starknet/modules/starknet_versions/pages/pathfinder_versions.adoc [hunk 1]
```diff
@@ -249,8 +249,12 @@ We added support for v0.3 and removed v0.1. We still support v0.2 at both `/rpc/
 We are missing `starknet_estimateFee` and `starknet_simulate` support for v0.3, which will be added in an upcoming release.
 
 ### Python requirement
+[NOTE]
+
+====
 
-Note: this only applies if you are building from source. This does not impact docker users.
+this only applies if you are building from source. This does not impact docker users.
```
Comment (stoobie) [inline components/Starknet/modules/starknet_versions/pages/pathfinder_versions.adoc:256]:
```suggestion
This only applies if you are building from source. This does not impact docker users.
```

File: components/Starknet/modules/tools/pages/bridged_tokens.adoc [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
-
-[horizontal, labelwidth="20"]
-`name`:: Token name.
-`symbol`:: Token symbol.
-`decimals`:: Number of decimal places used to get the user representation.
-`l1_token_address`:: Address of the L1 ERC-20 contract.
-`l2_token_address`:: Address of the L2 ERC-20 contract.
-`l1_bridge_address`:: Address of the L1 bridge contract.
-`l2_bridge_address`:: Address of the L2 bridge contract.
+*The tokens that are currently bridged to Starknet are listed in the following `.json` files:*
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/bridged_tokens.adoc:4]: Avoid formatting sentences with bold unless it's really necessary. We want to preserve markup language semantics, and using hard-coded formatting interferes with the page semantics.

If you need to make a sentence bold as a caption or title without adding structure via an actual heading, AsciiDoc provides dot headings:

`.A dot followed by text will be bold`

I use this syntax for the word *Procedure* to visually set off the beginning of a procedure.

In this case, I suggest keeping this sentence non-bold, and adding a title to the table by putting a doc heading with the title text on the line immediately preceding the beginning of the table.

File: components/Starknet/modules/tools/pages/bridged_tokens.adoc [hunk 3]
```diff
@@ -1,20 +1,33 @@
...
+| link:https://github.com/starknet-io/starknet-addresses/blob/master/bridged_tokens/mainnet.json[`mainnet.json`^]|  The addresses of the tokens currently bridged to Starknet Mainnet.
+| link:https://github.com/starknet-io/starknet-addresses/blob/master/bridged_tokens/goerli.json[`goerli.json`^] |The addresses of the tokens currently bridged to Starknet Goerli testnet.
+| link:https://github.com/starknet-io/starknet-addresses/blob/master/bridged_tokens/sepolia.json[`sepolia.json`^]
+
+| The addresses of the tokens currently bridged to Starknet Sepolia testnet.
+
+
+|===
+
+*Each token has the following parameters:*
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/bridged_tokens.adoc:19]:
```suggestion
.Token parameters
```

File: components/Starknet/modules/tools/pages/starkgate-cancelling a deposit.adoc [hunk 4]
```diff
@@ -16,7 +16,7 @@ In order to guard against an attack, it takes approximately five days to cancel
 
 . To cancel a deposit, call the `depositCancelRequest` request function.
 +
-When StarkGate receives the cancellation request, a counter begins to count five days.
+* When StarkGate receives the cancellation request, a counter begins to count five days.
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/starkgate-cancelling a deposit.adoc:19]: General rule: Never have a single list item. If you have one bullet, you need at least one more. If you only have on item, don't use a bullet.

Treat bullets and numbers as structural, not as formatting.

**Review by stoobie — CHANGES_REQUESTED** [review]
A few comments. Additionally, since I added you as a collaborator, you can add branches directly to the starknet-io instead of from your fork. That way you'll generate a preview so you can see what your changes will look like.

**Comment by stoobie** [comment]
@hardiktheprogrammer 
The default branch for this repo is now `main` instead of `master`. [Please follow these steps to update your local repos to use main instead of master from now on.](https://github.com/starknet-io/starknet-docs/discussions/964#discussioncomment-7982322)

**Comment by stoobie** [comment]
@hardiktheprogrammer Any update on this?

**Comment by stoobie** [comment]
@hardiktheprogrammer I'm closing this due to inactivity. Please feel free to repoen when you have time.

**Pull request closed without merging**